lib/src/main/kotlin/xyz/block/augur/FeeEstimator.kt

```diff
@@ -94,6 +94,35 @@ public class FeeEstimator @JvmOverloads public constructor(
     return convertToFeeEstimate(feeMatrix, orderedSnapshots.last().timestamp)
   }
 
+  public fun calculateEstimatesForNumberOfBlocks(
```

Comment:
if you can somehow do this in the `calculateEstimates` function instead of a new one that would be also good